src/lustre/lustreAbstractInterpretation.ml

```diff
@@ -503,6 +503,8 @@ and interpret_int_expr node_id ctx ty_ctx proj expr =
   | NArityOp _ -> assert false
   | ConvOp (_, _, e) -> interpret_int_expr node_id ctx ty_ctx proj e
   | CompOp _-> assert false
+  | ChooseOp (_, (_, _, ty), _) ->
+    extract_bounds_from_type ty
```

Comment:
This code should be unreachable since choose operators are desugared before this point. Replace it with `assert false` and comment.

Reply:
addressed